cmd/cdi/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/spf13/cobra v1.6.0
 	gopkg.in/yaml.v2 v2.4.0
 	sigs.k8s.io/yaml v1.3.0
-	tags.cncf.io/container-device-interface v0.0.0
+	tags.cncf.io/container-device-interface v1.0.0
 	tags.cncf.io/container-device-interface/schema v0.0.0
```

Comment:
Question: Do we want to also specify a version for this?

Reply:
reverted

Reply:
brougth both changes back as reverting them resulted in CI failures: https://github.com/cncf-tags/container-device-interface/actions/runs/13724737021